libs/service-portal/petitions/src/lib/messages.ts

```diff
@@ -110,6 +110,11 @@ export const m = defineMessages({
     defaultMessage: 'Sem CSV',
     description: '',
   },
+  linkToList: {
+    id: 'sp.petitions:linkToList',
+    defaultMessage: 'Hlekk á lista',
```

Comment:
```suggestion
    defaultMessage: 'Hlekkur á lista',
```

Reply:
þetta er í dropdown sem heitir Sækja
er það þá Hlekk eða hlekkur?

Reply:
Oh, didn't know the context. My bad, no "Hlekk" is correct.
I just assumed nominative :sweat_smile: